src/Components/Header.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+
+interface HeaderPropsType {
+  todos: Todo[],
+  searchQuery: string,
+  setSearchQuery: (searchQuery: string) => void,
+  addTodo: (title: string) => unknown,
+  onEmpty: () => void,
+  added: boolean,
```

Comment:
it would be better to use better naming as:
```suggestion
  addDisabled: boolean,
```